politeiawww/cmd/pictl/cmdcommentnew.go

```diff
@@ -117,5 +142,6 @@ Arguments:
                                  indicates that the comment is a reply.
 
```

Comment:
Needs better documentation.  What is an author update?  When are author updates allowed?  How does the user create an update?

Add to the description part of the help message (under the `If the record is unvetted, the --unvetted...` description).